stored_requests/events/files/files.go

```diff
@@ -0,0 +1,47 @@
+package files
+
+import (
+	"context"
+
+	"github.com/golang/glog"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/stored_requests"
+	"github.com/prebid/prebid-server/stored_requests/backends/file_fetcher"
+	"github.com/prebid/prebid-server/stored_requests/events"
+)
+
+type fileEventProducer struct {
+	invalidations chan events.Invalidation
+	saves         chan events.Save
+}
+
+func (f *fileEventProducer) Saves() <-chan events.Save {
+	return f.saves
+}
+func (f *fileEventProducer) Invalidations() <-chan events.Invalidation {
+	return f.invalidations
+}
+
+// NewFilesLoader returns an EventProducer preloaded with all the stored reqs+imps
+func NewFilesLoader(cfg config.FileFetcherConfig) events.EventProducer {
+	fp := &fileEventProducer{
+		saves:         make(chan events.Save, 1),
+		invalidations: make(chan events.Invalidation),
```

Comment:
Do we need the `invalidations` channel in the `fileEventProducer`? Doesn't look like we're using it

Reply:
It was there more for symmetry rather than anything functional. Will remove.